lib/utils.js

```diff
@@ -3,11 +3,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  *
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright 2020 Joyent, Inc.
  */
 
 module.exports = {
 	currentSerial: currentSerial,
+	downcaseKeys: downcaseKeys,
```

Comment:
Maybe call this `lowerCaseAllKeys` instead of `downcaseKeys` (i.e. use similar naming to `String.toLowerCase`).